sdks/python/apache_beam/runners/worker/statesampler.pyx

```diff
@@ -105,10 +109,12 @@ cdef class StateSampler(object):
   cdef pythread.PyThread_type_lock lock
 
   cdef public int64_t state_transition_count
+  cdef int64_t time_since_transition
```

Comment:
<!--new_thread; commit:3bc34eb5eaee80a75d69ba5df90bddc5f4b79980; resolved:0-->
Please initialize to 0 in `__init__`.